Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -304,73 +303,6 @@ public override long AppendIndex(Guid chainId, HashDigest<SHA256> hash)
             }
         }
 
-        /// <inheritdoc/>
-        public override IEnumerable<string> ListStateKeys(Guid chainId)
-        {
-            byte[] prefix = StateRefKeyPrefix;
-            var prevStateKey = string.Empty;
-
-            foreach (Iterator it in IterateDb(_stateRefDb, prefix, chainId))
-            {
-                byte[] key = it.Key();
-                int stateKeyLength = key.Length - sizeof(long) - prefix.Length;
-                byte[] stateKeyBytes = key.Skip(prefix.Length).Take(stateKeyLength).ToArray();
-                string stateKey = RocksDBStoreBitConverter.GetString(stateKeyBytes);
-
-                if (stateKey != prevStateKey)
-                {
-                    yield return stateKey;
-                    prevStateKey = stateKey;
-                }
-            }
-        }
-
-        /// <inheritdoc/>
-        public override IImmutableDictionary<string, IImmutableList<HashDigest<SHA256>>>
-            ListAllStateReferences(
-                Guid chainId,
-                long lowestIndex = 0,
-                long highestIndex = long.MaxValue)
-        {
-            byte[] prefix = StateRefKeyPrefix;
-
-            var stateRefs = new List<StateRef>();
-
-            foreach (Iterator it in IterateDb(_stateRefDb, prefix, chainId))
-            {
-                byte[] key = it.Key();
-                int stateKeyLength = key.Length - sizeof(long) - prefix.Length;
-                byte[] stateKeyBytes = key.Skip(prefix.Length).Take(stateKeyLength).ToArray();
-                string stateKey = RocksDBStoreBitConverter.GetString(stateKeyBytes);
-
-                byte[] indexBytes = key.Skip(prefix.Length + stateKeyLength).ToArray();
-                long index = RocksDBStoreBitConverter.ToInt64(indexBytes);
-
-                if (index < lowestIndex || index > highestIndex)
-                {
-                    continue;
-                }
-
-                var hash = new HashDigest<SHA256>(it.Value());
-                var stateRef = new StateRef
-                {
-                    StateKey = stateKey,
-                    BlockHash = hash,
-                    BlockIndex = index,
-                };
```

Comment:
If then, dosen't need `private class StateRef` anymore? I guess we need delete private class also.

Reply:
Thanks for letting me know the classes remained yet.

I removed all tracks of block states and state refs in f1c5e09d4365c52724ca571c68ec860097a72d7c. Please see with changed changelog 8fe90ef.